### PR TITLE
Feat: Add 'Get Attachments Email' Keyword

### PR DESCRIPTION
**Pull request by thuchet-u**
Here is a new Keyword to download all attachments of the given email_index

### CODE REVIEW

File: src/ImapLibrary2/__init__.py [hunk 1]
```diff
@@ -486,3 +487,36 @@ def _get_all_emails(self):
         """Saves all existing emails to internal variable."""
         typ, mails = self._imap.uid('search', None, 'ALL')
         self._mails = mails[0].split()
+
+    def get_attachments_email (self, **kwargs) :
```
Comment (lasselindqvist) [inline src/ImapLibrary2/__init__.py]: There are methods named
get_matches_from_email
and
get_links_from_email

We should probably name this get_attachments_from_email to be consistent.

Comment (thuchet-u) [reply src/ImapLibrary2/__init__.py]: Hi, it's done in my last commit

**Comment by thuchet-u** [comment]
Hi @lasselindqvist, thanks for the merge, have you planned to do a release ?

**Comment by lasselindqvist** [comment]
Hopefully this week.

**Comment by thuchet-u** [comment]
Hi @lasselindqvist, do you think you will have time to do the package soon ? Or should I do it another way ? Thanks

**Comment by lasselindqvist** [comment]
Sorry I just forgot it. Here it is now https://pypi.org/project/robotframework-imaplibrary2/0.4.6/

**Pull request merged**